sql/walrus--0.1.sql

```diff
@@ -194,7 +195,8 @@ $$;
 
 create type realtime.wal_column as (
     name text,
-    type text,
+    type_name text,
+    type_oid oid,
```

Comment:
@w3b6x9 Reminder to create the appropriate Realtime migration for these changes.